backend/src/main/java/com/jobwait/persistence/adapters/PostgresUserAdapter.java

```diff
@@ -1,17 +1,29 @@
 package com.jobwait.persistence.adapters;
 
+import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.UUID;
 
 import com.jobwait.domain.User;
 
-public class PostgresUserAdapter extends PersistenceAdapter<User>{
+public class PostgresUserAdapter extends PostgresAdapter<User> {
     public User fromResultSetRow(ResultSet rs) throws AdapterException {
         try {
             String id = rs.getString("id");
             UUID uuid = UUID.fromString(id);
-            return new User(uuid);
+
+            String authHash = rs.getString("authhash");
+
+            return new User(uuid, authHash);
```

Comment:
I guess this is technically a security risk because we might accidentally send the authHash to the user. Whatever, this is going to get replaced eventually when we write up the oauth2 handler